### PR TITLE
Todo Add, Remove Edit description

### PR DESCRIPTION
**Pull request by Nyame-Wolf**
In this pull request:
- I created a method to create todo items
- I created a button to delete a todo item and update index to match item removed
- i enabled editing of the description filed

### CODE REVIEW

File: src/modules/addRemoveEdit.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+};
+
+const refreshToDos = () => {
+  let child = ul.lastElementChild;
+  while (child) {
+    ul.removeChild(child);
+    child = ul.lastElementChild;
+  }
+  getToDos().forEach(createToDo);
+};
```
Comment (ishimwezachee) [inline src/modules/addRemoveEdit.js:56]: - [ ]  You should make this button delete all checked tasks. 
image

File: src/index.js [hunk 2]
```diff
@@ -1,45 +1,17 @@
...
+input.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    const completed = false;
+    const description = document.querySelector('.desc').value;
+    const index = getToDos().length + 1;
+    const newTodo = createToDo({ description, completed, index });
+    addToDo(newTodo);
+    input.value = '';
+  }
+});
```
Comment (ishimwezachee) [inline src/index.js:17]: - [ ]  When I delete one checked item, it is unchecking the other items please fix the issue.

File: src/index.js [hunk 3]
```diff
@@ -1,45 +1,17 @@
...
-  ul.appendChild(li);
-});
\ No newline at end of file
+getToDos().forEach(createToDo);
+input.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    const completed = false;
+    const description = document.querySelector('.desc').value;
+    const index = getToDos().length + 1;
+    const newTodo = createToDo({ description, completed, index });
```
Comment (ishimwezachee) [inline src/index.js:13]: - [ ]  You should fix the issue, that when you check the item, it should `completed `should be turned to `true`
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @Nyame-Wolf , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
The Project is running successfully 👍 
Well described PR 👍 
The Project is described professionally 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @Nyame-Wolf  👋,

_This is an additional review as a result of the second opinion request._

- The CR requested changes based on the project guidelines. So interactive-list milestone will come after the add-remove milestone.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by prolajumokeoni — APPROVED** [review]
Hi @Nyame-Wolf, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**